docs/vscode-extension/envchecker-help.md

```diff
@@ -80,7 +80,13 @@ It might be caused by timeout issue (longer than 3 minutes), the process to inst
 ### Mitigation
 Please refer to [the guide](#how-to-install-net-sdk) to install `.NET SDK` manually.
 
+## DotnetNotSupportTargetVersion
+> NETSDK1045: The current .NET SDK does not support 'newer version' as a target.
```

Comment:
keep consistent with the doc structure and add h3 like `Notification Message`?

Reply:
cool~, it's my fault